### PR TITLE
Update definitions for gay-torrents.net & gay-torrents.org

### PR DESCRIPTION
**Pull request by LukeSal88**
Adding additional functionality (sort/order, filters for freeleech/active/dead, etc)

### CODE REVIEW

File: src/Jackett.Common/Definitions/gay-torrents.yml [hunk 1]
```diff
@@ -8,6 +8,38 @@ encoding: UTF-8
...
+    default: 5
+    options:
+      0: published
+      1: title
+      2: files
+      4: size
+      5: seeders
+      6: leechers
+      7: downloads
+      8: health
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrents.yml:34]: limit to published, title, seeders, and size (and rename published to created)

File: src/Jackett.Common/Definitions/gay-torrents.yml [hunk 2]
```diff
@@ -80,9 +112,31 @@ login:
...
+        page: 4
+    - path: "{{ if .Keywords }}search.php{{ else }}torrentslist.php{{ end }}"
+      inputs:
+        page: 5
+    - path: "{{ if .Keywords }}search.php{{ else }}torrentslist.php{{ end }}"
+      inputs:
+        page: 6
+    - path: "{{ if .Keywords }}search.php{{ else }}torrentslist.php{{ end }}"
+      inputs:
+        page: 7
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrents.yml:134]: why are 7 pages necessary?

File: src/Jackett.Common/Definitions/gay-torrents.yml [hunk 3]
```diff
@@ -8,6 +8,38 @@ encoding: UTF-8
...
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Search freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort by
+    default: 5
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrents.yml:25]: default should be 0

File: src/Jackett.Common/Definitions/gay-torrentsorg.yml [hunk 4]
```diff
@@ -24,6 +24,36 @@ settings:
...
+    label: Status
+    default: 0
+    options:
+      0: All
+      1: Active Only
+      2: Dead Only
+      3: Free
+      4: 50%free
+      5: 25%free
+      6: packs
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrentsorg.yml:38]: change to 'search freeleech only' option

File: src/Jackett.Common/Definitions/gay-torrentsorg.yml [hunk 5]
```diff
@@ -88,7 +119,9 @@ search:
     $raw: "{{ range .Categories }}category[]={{.}}&{{end}}"
     search: "{{ .Keywords }}"
     # 0 both, 1 active, 2 dead
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrentsorg.yml:121]: update

File: src/Jackett.Common/Definitions/gay-torrentsorg.yml [hunk 6]
```diff
@@ -88,7 +119,9 @@ search:
...
-    active: 0
+    active: "{{ .Config.status }}"
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrentsorg.yml:122]: amend to work with 'search freeleech only' setting

File: src/Jackett.Common/Definitions/gay-torrentsorg.yml [hunk 7]
```diff
@@ -24,6 +24,36 @@ settings:
...
+    type: select
+    label: Sort by
+    default: seeds
+    options:
+      data: added
+      finished: downloads
+      seeds: seeders
+      leechers: leechers
+      size: size
+      filename: filename
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrentsorg.yml:49]: limit to added, filename, seeders, and size (and rename added to created, and filename to title)

File: src/Jackett.Common/Definitions/gay-torrentsorg.yml [hunk 8]
```diff
@@ -24,6 +24,36 @@ settings:
...
+      1: Active Only
+      2: Dead Only
+      3: Free
+      4: 50%free
+      5: 25%free
+      6: packs
+  - name: sort
+    type: select
+    label: Sort by
+    default: seeds
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/gay-torrentsorg.yml:42]: default should be data (is that correct, it isn't date?)

**Review by ilike2burnthing — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please see the requested changes below.

File: src/Jackett.Common/Definitions/gay-torrentsorg.yml (hunk 8 above)
Comment (garfield69) [reply src/Jackett.Common/Definitions/gay-torrentsorg.yml]: @LukeSal88  please confirm that `data` is the correct sort value for the date column

Comment (garfield69) [reply src/Jackett.Common/Definitions/gay-torrentsorg.yml]: on the basis that rockbox, nitro, p2pbg, pleasuredome and totallykids also have data as the date column sort parameter,  I'll merge this.

**Comment by garfield69** [comment]
v0.18.303

**Pull request merged**